src/main/java/wooteco/subway/controller/apis/StationController.java

```diff
@@ -0,0 +1,46 @@
+package wooteco.subway.controller.apis;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.domain.station.Station;
+import wooteco.subway.dto.StationRequest;
+import wooteco.subway.dto.StationResponse;
+import wooteco.subway.service.StationService;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/stations")
+public class StationController {
+
+    private final StationService stationService;
+
+    public StationController(StationService stationService) {
+        this.stationService = stationService;
+    }
+
+    @PostMapping
+    public ResponseEntity<StationResponse> createStation(@RequestBody StationRequest stationRequest) {
+        Station savedStation = stationService.createStation(stationRequest.getName());
+        StationResponse stationResponse = new StationResponse(savedStation.getId(), savedStation.getName());
+        return ResponseEntity.created(URI.create("/stations/" + savedStation.getId())).body(stationResponse);
+    }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<StationResponse>> showStations() {
+        List<StationResponse> stationResponses = stationService.findAll()
+                .stream()
+                .map(it -> new StationResponse(it.getId(), it.getName()))
+                .collect(Collectors.toList());
+        return ResponseEntity.ok().body(stationResponses);
```

Comment:
```suggestion
        return ResponseEntity.ok(stationResponses);
```

위와 같이도 가능하지 않나요 ㅎㅎ? 제가 지금 개발 환경이 없어서 틀릴 수도 있을 것 같아요 😉

Reply:
오... 그러네요. api를 더 꼼꼼하게 읽어봐야겠습니다. ㅠㅠ